content/es/02_programming-zkps-from-zero-to-hero.md

```diff
@@ -0,0 +1,752 @@
+---
+title: 'Programando ZKPs: De cero a héroe'
+date: '2025-07-23'
+tags: ['zero-knowledge']
+draft: false
+layout: PostSimple
+slug: "/es/programming-zkps-from-zero-to-hero"
+images: [../assets/02_combined.png']
+summary: "Aprende a escribir y modificar pruebas de conocimiento cero desde cero. Construirás un esquema de firma digital usando compromisos basados en hash, adquiriendo habilidades prácticas e intuición en programación de ZKP. Al final, tendrás todas las herramientas necesarias para implementar cosas como firmas grupales."
+---
+
+
```

Comment:
Can add a short italics line like this for translator:

<img width="533" height="101" alt="image" src="https://github.com/user-attachments/assets/fbb13d21-02b3-4b07-abdb-4d4036e00c2f" />